<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditGoalDialog</class>
 <widget class="QDialog" name="EditGoalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Goal</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>&amp;Title</string>
     </property>
     <property name="buddy">
      <cstring>edit_title</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_title"/>
   </item>
   <item>
    <widget class="QLabel" name="label_description">
     <property name="text">
      <string>&amp;Description</string>
     </property>
     <property name="buddy">
      <cstring>edit_description</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="edit_description"/>
   </item>
   <item>
    <widget class="QLabel" name="label_category">
     <property name="text">
      <string>&amp;Category</string>
     </property>
     <property name="buddy">
      <cstring>edit_category</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_category"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_progress">
       <property name="text">
        <string>&amp;Progress</string>
       </property>
       <property name="buddy">
        <cstring>edit_progress</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_goal">
       <property name="text">
        <string>&amp;Goal</string>
       </property>
       <property name="buddy">
        <cstring>edit_goal</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="edit_progress"/>
     </item>
     <item>
      <widget class="QLabel" name="label_outof">
       <property name="text">
        <string>/</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_goal"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="button_edit">
       <property name="text">
        <string>&amp;Edit</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Ca&amp;ncel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>EditGoalDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
